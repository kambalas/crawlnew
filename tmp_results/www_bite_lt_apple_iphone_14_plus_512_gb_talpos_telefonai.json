[
  {
    "name": "512GB Apple iPhone 14 Plus išmanusis telefonas 512 GB (PRODUCT)RED",
    "price": "1597,00 €",
    "error": false
  },
  {
    "name": "512GB Apple iPhone 14 Plus išmanusis telefonas 512 GB Yellow",
    "price": "1597,00 €",
    "error": false
  }
]