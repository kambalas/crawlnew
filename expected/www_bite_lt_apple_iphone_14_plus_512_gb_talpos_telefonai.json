[
  {
    "name": "Apple iPhone 14 Plus | BITĖ",
    "price": "1597.00",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Plus | BITĖ",
    "price": "1597.00",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Plus išmanusis telefonas 512 GB (PRODUCT)RED",
    "price": "1597",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Plus išmanusis telefonas 512 GB Yellow",
    "price": "",
    "error": false
  },
  {
    "index": 0,
    "error": true,
    "tags": [
      "error"
    ],
    "content": "'str' object has no attribute 'choices'"
  }
]